### ISSUE TITLE
Modifier Control Stick key only works when turning to the right 

### ISSUE THREAD

**Issue by DavideCannonito** [project-slippi/Ishiiruka#344]
Issue tracker is **ONLY** used for reporting bugs. Please use our [discord](https://slippi.gg/discord) for support and/or feature requests.

Hello, I have recently made the switch from controller to keyboard, as I was adapting in Training Mode I noticed that if I try to walk using the modifier button it works only on by turning to the right, if I try turning on the left the character starts to run. Note that the Modifier button works as expected on Dolphin 5.0-16101
## Expected Behavior

Modifier button should work on left and right
## Current Behavior

Only works on the right
## Steps to Reproduce

https://youtu.be/U2_2ZWL-X0k
1. Change input to keyboard
2. Set the Modifier bind to any key 
3. Open Melee and go to any game mode
edit: added youtube link showing this issue

## Environment
Windows 11

**Comment by ArcheCadia** [comment]
I have the exact same issue.

Slippi launcher:
Version 2.5.2 (ccee541)
Build 20220607
Windows 10

Faster Melee - Slippi (2.4.0)

**Comment by NikhilNarayana** [comment]
@JeremieHuteau @TelekomItalia would love if the two of you could test the PR above to see if it fixes the issue for you too.

**Comment by ArcheCadia** [comment]
The PR fixed the issue for me. I can walk to the left and to the right using modifiers. (Didn't test modifiers for the C-stick though, but should be ok too I guess)

**Comment by pdeblois** [comment]
Yeah the wording in my PR was a bit vague, but the issue was present for both the Gray Stick and C-Stick, and I fixed both since they work the same.

**Comment by Randanyo** [comment]
Casual keyboard player here, having the same issue falling offstage accidentally far too often. I'm not too experienced in this. Correct me if I'm wrong, but do I need to merge a pull request?  Can someone show me or link a tutorial on how to do that exactly, if they have the time? Thanks
